applications/RANSApplication/custom_strategies/rans_fractional_step_strategy.h

```diff
@@ -0,0 +1,306 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+//  Extended by :    Suneth Warnakulasuriya
+
+#if !defined(KRATOS_RANS_FRACTIONAL_STEP_STRATEGY)
+#define KRATOS_RANS_FRACTIONAL_STEP_STRATEGY
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_strategies/strategies/fractional_step_strategy.h"
+
+// Application includes
+
+namespace Kratos
+{
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @brief Fractional-step strategy for incompressible Navier-Stokes formulation
+ * This strategy implements a splitting scheme for the incompressible
+ * Navier-Stokes equations. It is intended to be used in combination with the
+ * FractionalStep element in the FluidDynamicsApplicatoin. The fractional step
+ * index, which is stored in the ProcessInfo, takes the values 1 : Momentum step
+ * (calculate fractional step velocity) 2-3 : Unused (reserved for componentwise
+ * calculation of frac step velocity) 4 : Pressure step 5 : Computation of
+ * projections 6 : End of step velocity
+ * @tparam TSparseSpace Sparse space template type
+ * @tparam TDenseSpace Dense space template type
+ * @tparam TLinearSolver Linear solver template type
+ */
```

Comment:
Would be nice to document in here why do we need a different FS strategy for the RANS element. As far as I can see both are pretty similar.

Reply:
It is, except for the lines
https://github.com/KratosMultiphysics/Kratos/blob/2aab494608c5549473d80be2179bec31650506b9/applications/RANSApplication/custom_strategies/rans_fractional_step_strategy.h#L206-L208

and

https://github.com/KratosMultiphysics/Kratos/blob/2aab494608c5549473d80be2179bec31650506b9/applications/RANSApplication/custom_strategies/rans_fractional_step_strategy.h#L215

Where I have to multiply pressure gradient with a pre-determined factor to acheive steady state solutions. It is based on this paper, 

"
A. R. Firoozjaee and M. H. Afshar, Steady-state solution of incompressible navier–stokes
equations using discrete least-squares meshless method, International Journal for Numerical
Methods in Fluids, 67 (2011), pp. 369–382.
".

I will add doxygen strings for all of these :)

Reply:
done :)